omnibus/config/software/msodbcsql18.rb

```diff
@@ -0,0 +1,50 @@
+name "msodbcsql18"
+default_version "18.3.2.1-1"
+
+dependency "unixodbc"
+
+skip_transitive_dependency_licensing true
+ship_source_offer true
+
+# Dynamically build source url and sha256 based on the linux platform and architecture
+# The source url and sha256 are taken from the official Microsoft ODBC Driver for SQL Server page:
+if debian_target?
+  if arm_target?
+    # Debian ARM64 build
+    package_name = "#{name}_#{version}_arm64.deb"
```

Comment:
I fear this lacks armv7 handling and will fail in some configurations, but the CI will likely confirm this

Reply:
We only build the IoT Agent on armv7 (which doesn't embed Python integrations like SQL Server), so this should be okay.